.github/workflows/smoke-tests.yaml

```
@@ -0,0 +1,80 @@
+name: consent-smoke-tests
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  smoke-tests:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: setup
+        id: setup
+        run:
+          echo "bee-name=${{ github.event.repository.name }}-${{ github.run_id }}-dev" >> $GITHUB_OUTPUT
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
```

```suggestion
          java-version: 17
          cache: 'maven'
```

I don't think we need the `actions/cache` step at all, the `setup-java` step should already cache all Maven outputs if you use the `cache: 'maven'` key: https://github.com/actions/setup-java#caching-maven-dependencies